src/lib/API/api.js

```diff
@@ -393,14 +393,14 @@ export class APIService {
   }
 
   // eslint-disable-next-line require-await
-  async notifyVendor(transactionId, vendorInfo) {
-    const { callbackUrl, invoiceId } = vendorInfo || {}
+  async notifyVendor(transactionId, transactionInfo) {
+    const { callbackUrl, invoiceId, senderEmail, senderName } = transactionInfo || {}
 
     if (!callbackUrl) {
       return // or throw error
     }
 
-    return this.client.post(callbackUrl, { invoiceId, transactionId })
+    return this.client.post(callbackUrl, { invoiceId, transactionId, senderEmail, senderName })
```

Comment:
were those fields names discussed & approved by @sirpy ?
was the server-side endpoint updated ? if not - a) who should do it b) have you tested - does it produces errors when you send some extra fields or everything is working fine ?

Reply:
Yes, I tested everything is working fine. There was already some completion when I started working on this, and those names are the ones specified in the task and also had already been used in the project before in places like `FeedStorage`:

```
  /**
   * in case of sending G$ directly, we keep tx details in an outbox so recipient can fetch it async
   * this is also used with the payment api that can add fields such as senderEmail,senderName,invoiceId
   * @param {*} event
   */
  async addToOutbox(event: FeedEvent) {
    let recipientPubkey = await this.userStorage.getUserProfilePublickey(event.data.to) //remove ~prefix

    if (recipientPubkey) {
      const pubKey = TextileCrypto.PublicKey.fromString(recipientPubkey)

      const data = pick(event.data, [
        'reason',
        'category',
        'amount',
        'senderEmail',
        'senderName',
        'invoiceId',
        'sellerWebsite',
        'sellerName',
      ])
```

Reply:
The server endpoint in this case is a callback specified by the vendor, the vendor is the one who should update his server.

Reply:
@TomasGonzalez `this.client` is an axios' instance configured to work with **our** server. it has `baseUrl` pre-set
please make sure urlyou passing is always absolute in this case

Reply:
What do you mean by `absolute`? I checked that the post was getting sent and it seem to be working fine, though nothing happened as the callbackUrl was a dummy one.

Reply:
https://some-sellers-site.com/some_callback_url

like this

Reply:
it shouldn't start just from /
does we have some validation for the seller's callback uri provided ?